src/docker/derby/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM eclipse-temurin:17-jre
+
+ENV DERBY_VERSION=10.16.1.1
+ENV DERBY_HOME=/derby
+ENV DERBY_LIB=${DERBY_HOME}/lib
+ENV CLASSPATH=${DERBY_LIB}/derby.jar:${DERBY_LIB}/derbynet.jar:${DERBY_LIB}/derbytools.jar:${DERBY_LIB}/derbyoptionaltools.jar:${DERBY_LIB}/derbyclient.jar
+
+RUN \
+    apt-get update && apt-get install -y wget netcat && \
+    wget https://dist.apache.org/repos/dist/release/db/derby/db-derby-${DERBY_VERSION}/db-derby-${DERBY_VERSION}-bin.tar.gz && \
```

Comment:
This is not a good idea, we may run into the rate limit issue we had with JMS connector, I think we should be uploading this tar.gz to nexus and downloading it with maven to be later used by docker. You can create a follow up ticket for this, as an outcome for this comment. But that change has to be performed eventually.

Reply:
Ok I will create the ticket

Reply:
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001h6g0VYAQ/view